bin/package.xml.in

```diff
@@ -67,5 +67,17 @@ necessary to build a fully-functional MongoDB driver.
   </required>
  </dependencies>
  <providesextension>mongodb</providesextension>
- <extsrcrelease/>
+ <extsrcrelease>
+  <configureoption name="enable-mongodb-developer-flags"       prompt="Enable developer flags? (yes/no)"                                            default="no" />
```

Comment:
Most other extensions omit printing valid values, but I think it's helpful since we have many that aren't strictly "yes/no" choices.